src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -584,10 +584,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         // Windows 10.
         // We'll also turn the acrylic back off when they're fully opaque, which
         // is what the Terminal did prior to 1.12.
-        if (!IsVintageOpacityAvailable())
-        {
-            _runtimeUseAcrylic = newOpacity < 1.0;
-        }
+        UseAcrylic(opacity < 1.0 && (!IsVintageOpacityAvailable() || _settings->UseAcrylic()));

```

Comment:
I'm not quite familiar with this code... Is the change from `_runtimeUseAcrylic` to `UseAcrylic()` correct?

Reply:
`UseAcrylic(...)` also updates the value of `_runtimeUseAcrylic`. I only changed it to be consistent with the way the opacity is updated a few lines above. Both are defined in a similar fashion.

https://github.com/microsoft/terminal/blob/e4b80e2ef33834a56f3c93582d0b87016aeb1bea/src/cascadia/TerminalControl/ControlCore.h#L35-L38
https://github.com/microsoft/terminal/blob/e4b80e2ef33834a56f3c93582d0b87016aeb1bea/src/cascadia/TerminalControl/ControlCore.h#L209